src/uglify/minify.js

```diff
@@ -40,7 +40,7 @@ const buildComments = (options, uglifyOptions, extractedComments) => {
   // /^\**!|@preserve|@license|@cc_on/
   if (typeof options.extractComments === 'boolean') {
     condition.preserve = commentsOpts;
-    condition.extract = /^\**!|@preserve|@license|@cc_on/;
+    condition.extract = /Copyright|LICENSE|@preserve|@license|@cc_on/ig;
```

Comment:
Don't remove `\**!`

Reply:
The regex won't work without it, since not everyone uses the `/**` type of comments.

Reply:
@evan-scott-zocdoc we use `|` (`or`) for comments types

Reply:
OHH, sorry I see. Will fix it.